nekoyume/Assets/_Scripts/UI/Widget/Screen/LoadingScreen.cs

```diff
@@ -76,6 +83,14 @@ protected override void OnDisable()
             base.OnDisable();
         }
 
+        public void ChangeToolTip()
+        {
+            if (_tips != null)
+            {
+                toolTip.text = _tips[new System.Random().Next(0, _tips.Count)];
```

Comment:
```suggestion
                toolTip.text = _tips[cachedRandom.Next(0, _tips.Count)];
```
I think it's inefficient to create a `System.Random` object every time.
Why don't you create a `System.Random` object, cache it, and call it up whenever random numbers are needed?

Reply:
I followed the original code, but I fixed it as you suggestion.😄 Can this be okay? Please refer to the changes.

Reply:
I'm not sure what code you're referring to, but I think it's a legacy code. This method continues to create objects that are used and discarded once and call GC.
Um... and It Looks like you didn't git push. I cann't check changes.